### PR TITLE
Improve contributors page

### PR DESCRIPTION
**Pull request by konifar**
* Added tap effect.
* Added number of contributors to title.
* Fixed GitHubAPI error handling.
* Change list order in about page.

### CODE REVIEW

File: app/src/main/java/io/github/droidkaigi/confsched/fragment/AboutFragment.java [hunk 1]
```diff
@@ -63,9 +63,6 @@ private void initView() {
         binding.imgTwitterClicker.setOnClickListener(v ->
                 AppUtil.showWebPage(getActivity(), AppUtil.getTwitterUrl(CONF_TWITTER_NAME)));
 
-        binding.txtTerms.setOnClickListener(v -> {
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched/fragment/AboutFragment.java]: It is not necessary.

File: app/src/main/res/layout/activity_contributors.xml [hunk 2]
```diff
@@ -13,21 +13,22 @@
             android:layout_width="match_parent"
             android:layout_height="?attr/actionBarSize"
             android:background="?attr/colorPrimary"
+            android:elevation="@dimen/elevation"
+            app:elevation="@dimen/elevation"
```
Comment (konifar) [inline app/src/main/res/layout/activity_contributors.xml]: Toolbar shadow.

File: app/src/main/java/io/github/droidkaigi/confsched/activity/ContributorsActivity.java [hunk 3]
```diff
@@ -94,9 +93,21 @@ private void renderSavedContributors(Throwable throwable) {
         Crashlytics.logException(throwable);
 
         List<Contributor> contributors = dao.findAll();
-        ContributorsAdapter contributorsAdapter = new ContributorsAdapter(this);
-        contributorsAdapter.addAll(contributors);
-        binding.recyclerView.setAdapter(contributorsAdapter);
+
+        if (contributors.isEmpty()) {
+            Snackbar.make(binding.getRoot(), R.string.contributors_load_error, Snackbar.LENGTH_LONG).show();
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched/activity/ContributorsActivity.java]: When GitHubAPI error occurred and there are no contributors data in DB, Show snackbar message.

File: app/src/main/java/io/github/droidkaigi/confsched/activity/ContributorsActivity.java [hunk 4]
```diff
@@ -94,9 +93,21 @@ private void renderSavedContributors(Throwable throwable) {
...
+        } else {
+            bindContributors(contributors);
+        }
+    }
+
+    private void bindContributors(List<Contributor> contributors) {
+        String title = binding.toolbar.getTitle().toString();
+        binding.toolbar.setTitle(title + getString(R.string.number_with_brackets, contributors.size()));
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched/activity/ContributorsActivity.java]: Add number of contributors to title.

2016-02-14_02_12_44_png

**Comment by gotokatsuya** [comment]
Thank you ! Awesome :100:

**Pull request merged**